tests/upgrades/test_subscription.py

```diff
@@ -121,7 +121,7 @@ def test_pre_subscription_scenario_autoattach(self, request):
         ).create()
         rhel7_client = dockerize(ak_name=act_key.name, distro='rhel7', org_label=org.label)
         client_container_id = [value for value in rhel7_client.values()][0]
-        ssh.command(
+        default_sat.execute(
```

Comment:
This command should get executed on the docker host, `hostname` was being passed to `ssh.command` here.

Reply:
good catch. just double-checked all other 121 references of `hostname=` in robottelo